usr/src/uts/armv8/sys/smp_impldefs.h

```diff
@@ -36,7 +37,7 @@
 extern "C" {
 #endif
 
-#define	WARM_RESET_VECTOR	0x467	/* the ROM/BIOS vector for 	*/
+#define	WARM_RESET_VECTOR	0x467	/* the ROM/BIOS vector for	*/
```

Comment:
This is copy/paste x86 code that Hayashi never removed, and I didn't spot until just now.  You can just delete it